pkg/printer/raw.go

```diff
@@ -0,0 +1,12 @@
+package printer
+
+import (
+	"fmt"
+	"io"
+)
+
+// PrintRaw is a printer that prints the provided value as is
```

Comment:
So raw is really a dump of the go structure? Let me know if I don't have that right.

What's the use case for this format? json and yaml I see people using that in scripts. If they are chaining the output of this new raw format into a Go program I want to make sure that maybe there isn't an easier way (like making it so that they can use our libraries and functions directly from their code). Or at least making sure that it's a consumable format (e.g. maybe we should be using `%#v` instead).

Also once we have hammered out the use case, let's pick a name that describes the format more clearly, e.g. "go" or something along those lines.

Reply:
Updated to change to `FormatPlaintext` per https://github.com/deislabs/porter-helm/pull/12#discussion_r254059588